pages/residences-parish.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
+    <div id="main-content" class="pl-5 pr-5 pt-3 pb-3">
+      <div class="flex content-start items-center">
+        <h1 class="text-sm md:text-xl lg:text-2xl">Number of Accused Witches That Resided In Each Modern Civil Parish</h1>
```

Comment:
We can remove the styling from this h1 now

Reply:
done